src/components/Album.vue

```diff
@@ -39,6 +49,17 @@ export default {
       .then(data => {this.album_info = data;
         this.is_loading = false
       })
+    },
+    artist_string(artists: {name: string}[]) {
+      let result = "";
+      for (let i = 0; i < artists.length - 1; i++){
+        result += artists[i].name + ", "
+      }
+      if (result != ""){
```

Comment:
Use of wrong operator, use "!==" so that it doesn't evaluate to true in falsey types.